packages/manager/apps/pci-public-ip/src/pages/additional-ips/edit/EditAdditionalIP.page.tsx

```diff
@@ -48,6 +48,9 @@ export default function EditInstancePage() {
         {(t) =>
           t('pci_additional_ips_failoverips_edit_error', {
             error: error?.response?.data?.message,
+            interpolation: {
+              escapeValue: false,
+            },
```

Comment:
we have this configuration directly in the initI18n function from react-shell-client package
It's better to not duplicate it in each translation

Reply:
The reason behind not make it in global config is the risk of XXS attack https://www.i18next.com/translation-function/interpolation#unescape

Reply:
Except we usually interpolate data that come from our APIs and display it to the user. We don't often interpolate data that come from user input and even less often send those to the APIs.

We have more cases where we don't want to escape value (because we use accents in our languages) than not doing it so it should be the default in my opinion